index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Asad Shareef Protfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="container">
+    <header>
+        <a href="#">My Logo</a>
+        <ul>
+            <li><a href="#">Protfolio</a> </li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <nav class="flex-nav"><button>☰</button></nav>
```

Comment:
I solved the issue and make the hamburger menu as shown in the Figma design.

Reply:
- Good job, @asadmisrani 👍
- [ ] Do you agree that a `dead link` is no link? I think you actually add your `index.html` as reference in the `href` attribute. This is improves access to your site and offers good user experience because visitors can always find their way back to your homepage in case they get lost somewhere and needed to confirm some info again.
- [ ] To make you hamburger button look like it is on the template, please consider removing the `button` tag so that the menu button is just in your `nav` tag.